.gitignore

```diff
@@ -18,3 +18,4 @@
 
 .DS_Store
 .env
+config/database.yml
```

Comment:
Alright. I still don't think you should add it to gitignore since it will affect everyone. Just don't commit it.